sdk/src/Handlers/AwsSdk/AWSSdkTracingHandler.cs

```diff
@@ -301,7 +301,7 @@ private void AfterResponseEventHandler(object sender, ResponseEventArgs e)
             var serviceName = RemoveAmazonPrefixFromServiceName(args.ServiceName);
             var operation = RemoveSuffix(args.Request.GetType().Name, "Request");
 
-            subsegment.Aws["region"] = client.Config.RegionEndpoint.SystemName;
+            subsegment.Aws["region"] = client.Config.RegionEndpoint?.SystemName;
```

Comment:
If you have a ServiceUrl and not a RegionEndpoint then you call Amazon.Utils.AWSSDKUtils.DetermineRegion() passing in the service url to get the region for it. That way we could keep the feature of tracing the region.

Reply:
Thanks for the suggestion @normj I will do a small PR to pick this up from the region, sounds reasonable.